kfp_notebook/pipeline/_notebook_op.py

```diff
@@ -75,7 +75,7 @@ def __init__(self,
         self.cos_directory = cos_directory
         self.cos_dependencies_archive = cos_dependencies_archive
         self.container_work_dir_root_path = "./"
-        self.container_work_dir_name = "jupyter-work-dir"
+        self.container_work_dir_name = "jupyter-work-dir/"
```

Comment:
Unrelated to this line, but could you please update the function's doc-string to include `pipeline_envs` and `emptydir_volume_size`?

Reply:
done